docs/user/dql/functions.rst

```diff
@@ -4299,6 +4299,28 @@ Another example to show how to set custom values for the optional parameters::
     | tEsT wIlDcArD sensitive cases             |
     +-------------------------------------------+
 
+NESTED
+------------
+
+Description
+>>>>>>>>>>>
+
+``nested(field | [field, path] | [path, condition])``
+
+The ``nested`` function maps to the ``nested`` query used in search engine. It returns nested field types in documents that match the provided specified field(s).
+The ``field`` and ``path`` parameters are valid parameters for a nested function used in the ``SELECT`` clause.
```

Comment:
Maybe reword
> The ``field`` and ``path`` parameters are valid parameters ...

What about `condition`? If it is not supported, remove it from syntax.
I propose mention that `nested` can be used in `SELECT` only in a separate sentence.